lib/modules/manager/bazel/update.spec.ts

```diff
@@ -119,25 +119,20 @@ describe('modules/manager/bazel/update', () => {
       const upgrade = {
         depName: 'com_github_google_uuid',
         depType: 'go_repository',
-        managerData: { def: input },
+        managerData: { idx: 0 },
         currentValue: 'v0.0.0',
         currentDigest,
         newDigest,
         newValue,
         updateType: 'major' as UpdateType,
       };
 
-      const output = input.replace(
-        `"${currentDigest}"`,
-        `"${newDigest}",  # ${newValue}`
-      );
+      const output = input.replace(`"${currentDigest}"`, `"${newDigest}"`);
 
       const res = await updateDependency({ fileContent: input, upgrade });
 
       expect(res).toEqual(output);
-      expect(res).toContain(
-        '"aaa09d789f3dba190787f8b4454c7d3c936fe123",  # v1.0.3'
```

Comment:
Do we really need this comment with tag? Seems like it's not worth the code complexity and problems this functionality may cause

Reply:
i think this is the same behavior which we use at GitHub actions to pin the git commit to a git tag.

so i think yes we need this.

Reply:
Here is the first time it was introduced: https://github.com/renovatebot/renovate/pull/3267

Reply:
so yes we need this

Reply:
I can confirm we've broken the solution to the original problem, so maintaining this comment with tag version makes no sense

Reply:
Let's focus on this problem later, I think I have better idea than tag commenting...